scripts/nolints.json

```
@@ -531,115 +531,23 @@
  ["docBlame", "Mathlib.Tactic.evalIntrov"],
- ["docBlame", "Mathlib.Tactic.expandExists"],
- ["docBlame", "Mathlib.Tactic.extractGoal!"],
- ["docBlame", "Mathlib.Tactic.failIfSuccess?"],
- ["docBlame", "Mathlib.Tactic.field"],
- ["docBlame", "Mathlib.Tactic.finish"],
- ["docBlame", "Mathlib.Tactic.fixingClause"],
- ["docBlame", "Mathlib.Tactic.generalizes"],
- ["docBlame", "Mathlib.Tactic.generalizesArg"],
- ["docBlame", "Mathlib.Tactic.generalizingClause"],
- ["docBlame", "Mathlib.Tactic.guardProofTerm"],
- ["docBlame", "Mathlib.Tactic.guardTags"],
- ["docBlame", "Mathlib.Tactic.hGeneralize"],
- ["docBlame", "Mathlib.Tactic.hGeneralize!"],
- ["docBlame", "Mathlib.Tactic.haveField"],
- ["docBlame", "Mathlib.Tactic.include"],
  ["docBlame", "Mathlib.Tactic.induction'"],
- ["docBlame", "Mathlib.Tactic.induction''"],
- ["docBlame", "Mathlib.Tactic.injectionsAndClear"],
- ["docBlame", "Mathlib.Tactic.interactive"],
- ["docBlame", "Mathlib.Tactic.intro"],
- ["docBlame", "Mathlib.Tactic.intro!"],
- ["docBlame", "Mathlib.Tactic.isBounded_default"],
- ["docBlame", "Mathlib.Tactic.listUnusedDecls"],
- ["docBlame", "Mathlib.Tactic.mapply"],
- ["docBlame", "Mathlib.Tactic.matchHyp"],
- ["docBlame", "Mathlib.Tactic.mkDecorations"],
- ["docBlame", "Mathlib.Tactic.mvBisim"],
- ["docBlame", "Mathlib.Tactic.notationClass"],
- ["docBlame", "Mathlib.Tactic.nthRwLHS"],
- ["docBlame", "Mathlib.Tactic.nthRwRHS"],
- ["docBlame", "Mathlib.Tactic.obviously"],
- ["docBlame", "Mathlib.Tactic.omit"],
- ["docBlame", "Mathlib.Tactic.padicIndexSimp"],
- ["docBlame", "Mathlib.Tactic.parameter"],
- ["docBlame", "Mathlib.Tactic.piInstance"],
- ["docBlame", "Mathlib.Tactic.piInstanceDeriveField"],
- ["docBlame", "Mathlib.Tactic.prettyCases"],
- ["docBlame", "Mathlib.Tactic.printSorryIn"],
- ["docBlame", "Mathlib.Tactic.propagateTags"],
- ["docBlame", "Mathlib.Tactic.protectProj"],
- ["docBlame", "Mathlib.Tactic.rcases?"],
- ["docBlame", "Mathlib.Tactic.reassoc"],
- ["docBlame", "Mathlib.Tactic.reassoc!"],
- ["docBlame", "Mathlib.Tactic.reassocAxiom"],
- ["docBlame", "Mathlib.Tactic.refineStruct"],
  ["docBlame", "Mathlib.Tactic.renameArg"],
- ["docBlame", "Mathlib.Tactic.revertAfter"],
- ["docBlame", "Mathlib.Tactic.revertDeps"],
- ["docBlame", "Mathlib.Tactic.revertTargetDeps"],
- ["docBlame", "Mathlib.Tactic.rintro?"],
- ["docBlame", "Mathlib.Tactic.rsimp"],
- ["docBlame", "Mathlib.Tactic.safe"],
- ["docBlame", "Mathlib.Tactic.sample"],
  ["docBlame", "Mathlib.Tactic.setArgsRest"],
  ["docBlame", "Mathlib.Tactic.setTactic"],
- ["docBlame", "Mathlib.Tactic.simpResult"],
- ["docBlame", "Mathlib.Tactic.subtypeInstance"],
- ["docBlame", "Mathlib.Tactic.suggest"],
- ["docBlame", "Mathlib.Tactic.tacticDestruct_"],
  ["docBlame", "Mathlib.Tactic.tacticHave_"],
  ["docBlame", "Mathlib.Tactic.tacticLet_"],
- ["docBlame", "Mathlib.Tactic.tacticMatch_target_"],
  ["docBlame", "Mathlib.Tactic.tacticSet!_"],
+ ["docBlame", "Mathlib.Tactic.tacticMatch_target_"],
```

Doesn't look very alphabetical here
```suggestion
 ["docBlame", "Mathlib.Tactic.tacticMatch_target_"],
 ["docBlame", "Mathlib.Tactic.tacticSet!_"],
```